lib/commons/dom/get-visible-child-text-rects.js

```diff
@@ -0,0 +1,80 @@
+import { getNodeFromTree, memoize } from '../../core/utils';
+import { sanitize } from '../text';
+import { getRectCenter, isPointInRect, getIntersectionRect } from '../math';
+import getOverflowHiddenAncestors from './get-overflow-hidden-ancestors';
+
+/**
+ * Get the visible text client rects of a node.
+ * @method getVisibleChildTextRects
+ * @memberof axe.commons.dom
+ * @instance
+ * @param {Element} node
+ */
+const getVisibleChildTextRects = memoize(
+  function getVisibleChildTextRectsMemoized(node) {
```

Comment:
Should this method handle CSS clip, in addition to overflow:hidden? If so I think a separate issue might be in order.

Reply:
Possibly. We don't have any tests around this so a separate ticket would be good.

Reply:
Clip is deprecated, skipping.